tests/switchmap_/poller/test_poll.py

```
@@ -0,0 +1,187 @@
+        with patch("switchmap.poller.poll.ConfigPoller") as mock_config:
+            mock_config.return_value = self.mock_config_instance
+            with patch("switchmap.poller.poll.Pool") as mock_pool:
+                mock_pool_instance = MagicMock()
+                mock_pool.return_value.__enter__.return_value = (
+                    mock_pool_instance
+                )
+                mock_pool_instance.map.side_effect = Exception(
+                    "Worker process failed"
+                )
+
+                with self.assertRaises(Exception):
```

Ruff (0.8.2)

`assertRaises(Exception)` should be considered evil

(B017)